mypyc/test/test_namegen.py

```diff
@@ -7,33 +7,34 @@
 
 class TestNameGen(unittest.TestCase):
     def test_candidate_suffixes(self) -> None:
-        assert candidate_suffixes('foo') == ['', 'foo_']
-        assert candidate_suffixes('foo.bar') == ['', 'bar_', 'foo_bar_']
+        assert candidate_suffixes('foo') == ['', 'foo.']
```

Comment:
Would it be useful to have an end-to-end test case that has things like triple underscores in names and possible name clashes, such as having names `C.foo` and `C___foo` in the same namespace?

Reply:
I'll add that to the name clash issue test